From cc96162a47bd88ba76cece5043a93b438e48c2c7 Mon Sep 17 00:00:00 2001
From: Jeffrey Stedfast <fejj@gnome.org>
Date: Sat, 02 Nov 2013 20:08:52 +0000
Subject: Don't write out extra spaces in the References header

2013-11-02  Jeffrey Stedfast  <fejj@gnome.org>

	* gmime/gmime-message.c (write_references): Don't write out 2
	spaces between the ':' and the first message-id. Fixes bug
	#711305.
---
diff --git a/gmime/gmime-message.c b/gmime/gmime-message.c
index b6243fd..3b00ee1 100644
--- a/gmime/gmime-message.c
+++ b/gmime/gmime-message.c
@@ -636,7 +636,7 @@ write_references (GMimeStream *stream, const char *name, const char *value)
 	   it seems to break a lot of clients (and servers) */
 	references = g_mime_references_decode (value);
 	folded = g_string_new (name);
-	g_string_append_len (folded, ": ", 2);
+	g_string_append_c (folded, ':');
 	len = folded->len;
 	
 	reference = references;
--
cgit v0.9.2
